client/src/styles/forms.scss

```diff
@@ -0,0 +1,82 @@
+$cms-border: #ced5e1;
+$cms-text-darkbg: white;
+$cms-danger: #d40404;
+$cms-button: #008a00;
+$cms-button-dim: darken($cms-button, 10%);
+
+fieldset {
+  border: 0;
+  padding: 0;
+  margin: 0;
+}
+
+.field {
+  margin-bottom: 1em;
+}
+
+.text input {
+  font-size: 1em;
+  padding: 0.7em;
+}
+
+.middleColumn { // sass-lint:disable-line class-name-format
+  margin: 0.2em 0;
+}
+
+.field.message {
+  font-size: 0.8em;
+  margin-top: 0.5em;
+  display: block;
+}
+
+.message.required {
```

Comment:
Note that this won't be required in SilverStripe 4.4 onwards, since it uses Bootstrap alert classes

Reply:
I considered we could (perhaps should) introduce form field templates to the theme so a consistent style can be ensured no matter if a dev should introduce overrides elsewhere.
But then as this is a theme any template added to the project module would still override them. The value is low, I figure we should cross that bridge when we come to it (in another issue).

The up side of providing templates is that the interface will no longer depend on core settings (i.e. the module will be compatible with older versions of 4.x and versions newer than or equal to 4.4).